google-cloud-logging/lib/google/cloud/logging/logger.rb

```diff
@@ -430,7 +485,7 @@ def reopen _logdev = nil
         # @deprecated Use add_request_info
         #
         def add_trace_id trace_id
-          add_request_id trace_id: trace_id
+          add_request_info trace_id: trace_id
```

Comment:
I assume this change is correct. The method `add_request_id` doesn't exist AFAICT.

Reply:
Yes you're right. 😱